inflate_fast_amd64_audit_test.go

```
@@ -0,0 +1,435 @@
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			d, r := tt.prepare(t)
+			// Large enough that the kernel, not the tail, must detect
+			// each site, with room for the 32 KiB history the reserved
+			// cases decode first.
+			const dstLen = 32768 + 2*deflateFastOutputMargin
+
+			reference := amd64RunHuffmanGo(d, r, dstLen)
+			if reference.err != errDeflateBadData {
```

golangci-lint (2.12.2)

[error] comparing with != will fail on wrapped errors. Use errors.Is to check for a specific error

(errorlint)